ci-operator/step-registry/ipi/conf/vsphere/ipi-conf-vsphere-ref.yaml

```diff
@@ -7,6 +7,10 @@ ref:
       cpu: 10m
       memory: 100Mi
   env:
+  - name: IPAM_VERSION
+    default: ""
```

Comment:
What happens when it's the empty string?

Reply:
In the script, it defaults to "LATEST".  If empty ends up making its way to the template, the default there is LATEST as well.